crmlogin/addField.php

```diff
@@ -0,0 +1,121 @@
+<?php 
+	include 'connection.php';
+	include 'Contact.php';
+	session_start();
+	$user = $_SESSION['user'];
+	if(!isset($user)){
+		header("Location: index.html");
+	}
+	
+
+	// $sqll = "SELECT COLUMN_NAME, DATA_TYPE, IS_NULLABLE, COLUMN_DEFAULT FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'contactlist'";
+	$sqll = "SELECT `field` FROM `fieldrelation` WHERE `user` = '$user'";
+  	$resultt = $conn->query($sqll);
+	$i = 0;
+	$colNames = [];
+    while($row = $resultt->fetch_assoc()) {
+    	$fieldid = $row['field'];
+    	$sql = "SELECT `fieldname` FROM `fields` WHERE `fieldid` = '$fieldid'";
+    	$result = $conn->query($sql);
+    	$fieldname = $result->fetch_assoc();
+    	$colNames[$i] = $fieldname['fieldname'];
+    	// $colNames[$i] = $row['COLUMN_NAME'];
+		// echo "<p>".$row['COLUMN_NAME']."</p>";
+    // echo "<p>".$row[$i]."</p>";
+    	$i++;
+	}
+	$colList = "";
+	for($i = 0; $i < count($colNames); $i++){
+		// $colList .= "<tr><td style='background-color:grey'><input name='checkbox[]' type='checkbox' value = ".$colNames[$i]."></td>";
+		$colList .= "<td style='background-color: grey ;color:black;font-size: 20px'>".$colNames[$i]."</td></tr>";
+		// echo $colNames[$i];
+	}
+
+	// $sql = "SELECT * FROM `contactlist` WHERE `user`='$user'";

```

Comment:
Don't pepper code with comments, unless its used for some later testing, remember its in the repo so you can always look through the history to get.